src/main/java/alibek/ChatServer.java

```diff
@@ -0,0 +1,76 @@
+package alibek;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.*;
+
+
+public class ChatServer {
+    ArrayList clientOutputStreams;
+    private ServerSocket serverSocket;
+    public Map<String, User> userHashMap;
```

Comment:
&#x1F34E; **Issue**
You don't really need to make this field public.
Also, check every other access modifier and make sure they are the most consealing